fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ColumnRefOperator.java

```diff
@@ -19,11 +19,22 @@ public final class ColumnRefOperator extends ScalarOperator {
     private final String name;
     private boolean nullable;
 
+    private boolean isLambdaArgument;
```

Comment:
I think we needn't add a isLambdaArgument field to ColumnRefOperator

Reply:
https://github.com/fzhedu/starrocks/blob/ce0a127840fec341b4dfaf9ded1a4f5f3bc29916/fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ColumnRefOperator.java#L82-L87

not yet, it marks that lambda arugments should not regared as used columns when out of the lambda function.